src/status_im/chat/models/message.cljs

```diff
@@ -148,10 +148,13 @@
             (chat-loading/group-chat-messages chat-id (get chat->message chat-id))))
 
 (defn- add-to-chat?
-  [{:keys [db]} {:keys [chat-id clock-value message-id] :as message}]
+  [{:keys [db]} {:keys [chat-id clock-value message-id message-id-old-format] :as message}]
   (let [{:keys [deleted-at-clock-value messages not-loaded-message-ids]}
         (get-in db [:chats chat-id])]
     (not (or (get messages message-id)
+             ;; TODO(rasom): remove this condition
```

Comment:
Yeah, that's true, that user will encounter this problem. 

No decisions were made, my opinion is that it is ok. But we also can keep this condition for all period while 0.9.29 will be available, for TestFlight it is 77 days from now and idk how to check this for Play Market.

Another option might be a migration for ids, but it might cause even more bugs in the result.

Reply:
I would agree that is not a big deal, and seems ok to just leave the code for a bit longer.
If we do go for a migration though I think we should re-think the id generation, as the current method as the same drawbacks as the previous (namespace included / relies on consistent ordering of map keys). (maybe even worth doing now, but I understand there some time pressure).

Reply:
yep, there is definitely some pressure, i suppose we will just return to this again later...

Reply:
@cammellos, i changed comment a bit